File: Tfecha.cpp
class Tfecha
{
public:// Parte pública de la clase
TFecha( ); //Constructor por defecto
TFecha(int,int,int ); //Constructor sobrecargado
TFecha(TFecha&); //Constructor de copia
~TFecha( ); //Destructor

};
TFecha::TFecha(int d, int m, int a)
{
dia = d; //O bien, AsignarDia(d);
mes = m; //O bien, AsignarMes(m);
anyo = a; //O bien, AsignarAnyo(a);
}

File: TFecha.py
class tfecha(object):

	def __init__(self, dia, mes, anyo): #definir variables
		self.dia = dia
		self.mes = mes
		self.anyo = anyo

	def esbisiesto(self):	
		if self.anyo % 4 == 0 and (self.anyo % 100 != 0 or anyo % 400 == 0):
			print("Es bisiesto")
		else:
			print("No es bisiesto")